### PR TITLE
Additional gherkin scenarios for the options.ini file

### PR DESCRIPTION
**Pull request by pcenov**

### CODE REVIEW

**Comment by pcenov** [comment]
@radinamatic

File: integration_testing/features/super-admin/commands/super-admin-customizes-kolibri-settings.feature [hunk 1]
```diff
@@ -27,10 +27,10 @@ Feature: Super admin customizes the Kolibri settings with the options.ini file
     Then I can see the content moved in the destination folder
 
   Scenario: Change the default central content base URL
-    When I add "CENTRAL_CONTENT_BASE_URL = <content base URL> under the [Urls] section
+    When I add "CENTRAL_CONTENT_BASE_URL = <content base URL>" under the [Urls] section
       And I save my changes
       And I run the "kolibri start" command in the terminal or command prompt
-      And I go to Device>Channels
+      And I go to *Device>Channels*
```
Comment (radinamatic) [inline integration_testing/features/super-admin/commands/super-admin-customizes-kolibri-settings.feature]: We usually separate this so, ``Device > Channels``, right?

File: integration_testing/features/super-admin/commands/super-admin-customizes-kolibri-settings.feature [hunk 2]
```diff
@@ -55,6 +55,26 @@ Feature: Super admin customizes the Kolibri settings with the options.ini file
     When the <facility> is done syncing
       Then I see a message under the <facility> *Last synced: just now*
 
+  Scenario: Enable custom channel navigation
+    When I add "ENABLE_CUSTOM_CHANNEL_NAV = True" under the [Learn] section
+      And I save my changes
+      And I run the "kolibri start" command in the terminal or command prompt
+      And I go to *Device>Channels*
```
Comment (radinamatic) [inline integration_testing/features/super-admin/commands/super-admin-customizes-kolibri-settings.feature]: Idem for separation here.

**Comment by radinamatic** [comment]
Looking good so far, thank you! Let me now if there's still something pending, otherwise I'll go ahead and merge 👍🏽

**Review by radinamatic — APPROVED** [review]
Good to have these checked and updated, thank you! 👍🏽 💯

**Pull request merged**